Samples/Islands/DrawingIsland/DrawingCppTestApp/readme.txt

```diff
@@ -0,0 +1,30 @@
+========================================================================
+    C++/WinRT DrawingIsland Project Overview
+========================================================================
+
+This project demonstrates how to get started consuming Windows Runtime 
+classes directly from standard C++, using platform projection headers
+generated from Windows SDK metadata files.
```

Comment:
Can this file either be deleted, or converted to a .md explaining what the test app is doing and why?